### DISCUSSION TITLE
HA log spam when restarting AP the r-a is connected to

### DISCUSSION THREAD

**Discussion by hugalafutro** [mKeRix/room-assistant#807] [Q&A] [answered]
as the title says, I was updating my mesh ap firmware and noticed 36 error messages in HA log within a minute while the device room-assistant runs on was offline. 
```
Logger: homeassistant.util.logging
Source: util/logging.py:105
First occurred: 17:50:42 (36 occurrences)
Last logged: 17:50:43

Exception in availability_message_received when handling msg on 'room-assistant/status/home-bedroom-table': 'online' Traceback (most recent call last): File "/usr/src/homeassistant/homeassistant/components/mqtt/debug_info.py", line 32, in wrapper msg_callback(msg) File "/usr/src/homeassistant/homeassistant/components/mqtt/mixins.py", line 323, in availability_message_received if msg.payload == self._avail_topics[topic][CONF_PAYLOAD_AVAILABLE]: KeyError: 'room-assistant/status/home-bedroom-table'
```
not really sure if it's r-a issue or ha issue (let me know if it is I'll take this to HA forums) or if it's even an issue - just wondering.

Thanks for any info!

**Comment by mKeRix** [comment, marked as answer]
I've seen this as well, it shouldn't impact functionality in general because it fixes itself fairly quickly. Has to do with how the availability logic works in Home Assistant when using multiple availability topics as Home Assistant does.
